src/components/reports/TableScoreTag.vue

```diff
@@ -5,7 +5,7 @@
   >
     <PvTag
       :value="_get(colData, col.field)"
-      :style="`background-color: ${_get(colData, col.tagColor)}; min-width: 2rem; 
+      :style="`background-color: #74797f; min-width: 2rem; 
```

Comment:
```suggestion
      :style="`background-color: ${_get(colData, col.tagColor) ? _get(colData, col.tagColor) : '#74797f'}; min-width: 2rem; 
```
Similar to Adam's suggestion, do we want to make this a fallback value rather than as a hard coded value?